test/unit/validators/base.spec.js

```diff
@@ -90,7 +90,7 @@ describe("Test BaseValidator 'middleware' method", () => {
 			// Call wrapped function
 			return wrapped(ctx).then(() => {
 				expect(checkGood).toHaveBeenCalledTimes(1);
-				expect(checkGood).toHaveBeenCalledWith({ "id": 5, "name": "John" });
```

Comment:
Please leave the full object def `{ "id": 5, "name": "John" }`. It's a kind of protection checking the params whether is manipulated. The `ctx.params` hide it.